pkgs/servers/clickhouse/default.nix

```diff
@@ -29,13 +29,6 @@ stdenv.mkDerivation rec {
     "-DUSE_INTERNAL_SSL_LIBRARY=False"
```

Comment:
Please remove `-DUSE_INTERNAL_SSL_LIBRARY=False`, it is implied by `-DUNBUNDLED=ON`.

Reply:
done